data_store/cli.py

```diff
@@ -40,6 +41,7 @@ def seed_reference():
         with current_app.app_context():
             seed_geospatial_dim_table()
             seed_fund_table()
+            load_example_data()
```

Comment:
Nice fix! I would say there's probably value in having the `load_example_data()` get called in a separate CLI command so that this one stays distinct to just seeding reference data (and then updating the docker-runner to call both on start-up). There might be scenarios where we want to seed or update just the reference dat.

Reply:
Started making the change, but now I notice that `flask db-data reset` populates back only reference data.

Hmm.. there are a number of ways to go about it:
1. Change `seed_reference` to `seed_data` and have it do both ref and data seeding. Make reset add both back.
2. Make command accept a parameter if it should populate/reset sample data with the ref
3. Duplicate commands. Separate seeding commands for ref/data and reset ref/data.

Thoughts?

Reply:
My preference would be for option 3. Reasoning being:

- Keeps things separate - if we want to seed, we have commands to do it, and if we want to reset, we have commands to do it.
- Leaves `flask db-data reset` as is, as I think the intention with that command was to reset the DB into a useable state (ie. empty but with the necessary reference data seeded so you could start to ingest spreadsheets). I don't know if reset should also seed non-ref data, but happy to be convinced otherwise - logically it makes sense to me that if you wanted to do that you'd reset and then run whatever new seed data command you add.